packages/noco-docs/docs/080.records/030.expand-record.md

```diff
@@ -57,6 +57,9 @@ The Comments feature allows you to collaborate directly within records by leavin
     <iframe width="560" height="315" src="https://www.youtube.com/embed/U5ZYVlpOGN8?start=12" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen ></iframe>
 </center>
 
+:::info
+This is only supported in the Cloud Version
+:::
```

Comment:
Thank you for your contribution to the documentation! I'd like to suggest a small but important correction regarding feature availability:

The expanded record view itself is actually available in all versions of NocoDB. Only the Mentions, and Resolve features within the comments are exclusive to Cloud/Enterprise Edition.

Would you mind updating the documentation to be more specific? Instead of marking the entire expand-record as Cloud-only